app/ui-message/client/message.html

```diff
@@ -32,7 +32,7 @@
 			<span class="time" title='{{date}} {{time}}'>{{time}}</span>
 			{{#if showTranslated}}
 				<span class="translated">
-					<i class="icon-language {{#if autoTranslateFetching}}loading{{/if}}" aria-label="{{_ "Translated"}}"></i>
+					<i class="icon-language {{#if msg.autoTranslateFetching}}loading{{/if}}" aria-label="{{_ "Translated"}}"></i>
```

Comment:
is `msg.autoTranslateFetching` already available in this branch?

Reply:
ops